sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/BackupResource.cs

```diff
@@ -0,0 +1,153 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+using Azure.ResourceManager;
+
+namespace Azure.ResourceManager.CosmosDB
+{
+    /// <summary>
+    /// A Class representing a BackupResource along with the instance operations that can be performed on it.
+    /// If you have a <see cref="ResourceIdentifier" /> you can construct a <see cref="BackupResource" />
+    /// from an instance of <see cref="ArmClient" /> using the GetBackupResource method.
+    /// Otherwise you can get one from its parent resource <see cref="CassandraClusterResource" /> using the GetBackupResource method.
+    /// </summary>
+    public partial class BackupResource : ArmResource
```

Comment:
`BackupResource` is too generic, consider renaming it to `CassandraClusterBackupResource`.